src/components/conditional.js

```diff
@@ -70,6 +74,24 @@
         B.defineFunction('Set Visibility', value => {
           if (typeof value === 'boolean') setVisible(value);
         });
+        B.defineFunction('Set Left Value', value => {
+          if (typeof value === 'object') {
+            value.length == undefined
+              ? setLeftValue(value.target.value)
+              : setLeftValue(value.map(String));
+          } else {
+            setLeftValue(value.toString());
+          }
+        });
+        B.defineFunction('Set Right Value', value => {
+          if (typeof value === 'object') {
+            value.length == undefined
+              ? setRightValue(value.target.value)
+              : setRightValue(value.map(String));
+          } else {
+            setRightValue(value.toString());
+          }
+        });
```

Comment:
You're doing exactly the same twice, try to prevent duplicate code by extracting the same logic in a separate function: 
```
const getValue = evt => {
          if (typeof evt !== 'object') {
            return evt.toString();
          }
          if (evt.length !== undefined) {
            return evt.map(String);
          }
          if (evt.target && !evt.target.checked) {
            return [];
          }
          return evt.target && evt.target.value;
        };

        B.defineFunction('Set Left Value', event =>
          setLeftValue(getValue(event)),
        );
        B.defineFunction('Set Right Value', event =>
          setRightValue(getValue(event)),
        );
```

Also you can use `yarn lint` after making your changes to check if you have any errors before you open a PR and do this via the pipeline.